data/remnant/remnant 1 introduction.txt

```diff
@@ -150,7 +150,7 @@ mission "Remnant Hails Change"
 
 
 
-mission "lost in ember waste"
+mission "Remnant: lost in ember waste"
```

Comment:
Don't change mission identifiers after they've been released unless absolutely necessary.

Everyone who already experienced the mission will be eligible for it again.

Reply:
As it stands, this is one of the least-seen missions in the history of the game. Someone accidentally getting it a second time is only likely to happen if someone *tries* to get it. But sure, I can fix this.

Reply:
Has now been fixed. People cannot get this mission a second time.